cmd/flow-access-server/main.go

```diff
@@ -57,11 +57,13 @@ func run() int {
 
 	// Command line parameter initialization.
 	var (
+		flagChain string
 		flagLevel string
 		flagIndex string
 		flagPort  uint16
 	)
 
+	pflag.StringVarP(&flagChain, "chain", "c", dps.FlowMainnet.String(), "ID of the chain that this server uses as its data source")
```

Comment:
I wonder what we should do about this. It should not be possible to set the wrong chain ID for the server. So in a way it should be persisted in the index DB. However, the indexer is agnostic of the chain ID. I think we leave it like this for now, but let's create an issue to investigate if we can deduce the chain ID from the protocol state or the execution state in the future.